lib/bora/stack.rb

```diff
@@ -180,26 +179,28 @@ def diff_parameters(cfn_options)
 
       current_params_str = params_as_string(current_params)
       new_params_str = params_as_string(new_params)
-      if current_params_str || new_params_str
-        puts "Parameters".colorize(mode: :bold)
-        puts "----------"
-        diff = Diffy::Diff.new(current_params_str, new_params_str).to_s(String.disable_colorization ? :text : :color).chomp
-        puts diff && !diff.empty? ? diff : STACK_DIFF_PARAMETERS_UNCHANGED_MESSAGE
-        puts
-      end
+      return unless current_params_str || new_params_str
+      puts 'Parameters'.colorize(mode: :bold)
+      puts '----------'
+      diff = Diffy::Diff.new(current_params_str, new_params_str).to_s(String.disable_colorization ? :text : :color).chomp
+      # TODO: False positive?
+      # rubocop:disable  Lint/RequireParentheses
+      puts diff && !diff.empty? ? diff : STACK_DIFF_PARAMETERS_UNCHANGED_MESSAGE
+      # rubocop:enable Lint/RequireParentheses
```

Comment:
Should resolve the TODO before merging. To be honest this expression is complicated and probably warrants refactoring. Maybe something like:
```ruby
unchanged = diff.nil? || diff.empty?
puts unchanged ? STACK_DIFF_PARAMETERS_UNCHANGED_MESSAGE : diff
```